vlasiator.cpp

```diff
@@ -71,6 +71,7 @@ void fpehandler(int sig_num)
 
 Logger logFile, diagnostic;
 static dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry> mpiGrid;
+SysBoundary sysBoundaryWrapper;
```

Comment:
Instead of placing this here as a global variable, another option would be to move it into the objectWrapper. That way, we are not further polluting the global variable namespace and have a little bit of a better encapsulation of global simulation state.

Reply:
That would indeed be an alternative option. I'l look into it.